app/models/person_escort_record.rb

```diff
@@ -2,6 +2,8 @@ class PersonEscortRecord < VersionedModel
   include FrameworkAssessmentable
 
   belongs_to :prefill_source, class_name: 'PersonEscortRecord', optional: true
+  # To support legacy PERs without a move, allow the association to be optional
+  belongs_to :move, optional: true
```

Comment:
Is there a plan to clean up legacy data at some point... maybe with a rake task that finds a move with a date around the same per created_at date perhaps? Would be nice to not have to think about two different possible relationships between PER, profile and move to reduce future cognitive load.

Reply:
That's definitely a good idea. These issues weren't prominent previously, but I think now we might need to do it, to avoid all those special cases and complication in supporting the legacy PERs. I'll add tickets for this to be tackled soon 👍